scripts/models/repo.js

```diff
@@ -0,0 +1,21 @@
+
+var reposObj = {};
+
+reposObj.all = [];
+
+reposObj.requestPushRepos = function(){
+  $.ajax({
+    url: 'https://api.github.com/users/TomTimmel/repos',
```

Comment:
You'll also want to add in a headers property here to feed in your github token. Something like:

```        headers: {
          Authorization: 'token ' + token
        } ```